packages/url_launcher/url_launcher_web/example/run_test

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/bash
+if pgrep -lf chromedriver > /dev/null; then
```

Comment:
nit: maybe specify how to use this script here?  Also, what about adding the extension `.sh`?

Reply:
I'm going to send people to the README.md if it fails the chromedriver check :P